tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
     sphinx
     sphinx-rtd-theme
     -rdev-requirements.txt
-commands=make html
+commands=/usr/bin/make html
 
```

Comment:
Agreed, this was an oversight on my part. I didn't mean to modify tox.ini at all